src/helix-ui/styles/elements/hx-tile.less

```diff
@@ -1,5 +1,7 @@
+@import (reference) 'components/box';
+
 hx-tile {
-  &:extend(.hxBox-lg);
+  #Box.md();
```

Comment:
(CHANGED) Default padding of `<hx-tile>` dropped from `1.5rem/24px` to `1.25rem/20px` to align with standard spacing specifications.